src/components/CreateNewProjectWizard/MainPane.js

```diff
@@ -30,7 +30,7 @@ type Props = {
   isProjectNameTaken: boolean,
   updateFieldValue: (field: Field, value: any) => void,
   focusField: (field: ?Field) => void,
-  handleSubmit: () => void,
+  handleSubmit: () => Promise<any> | void,
```

Comment:
Interesting, I wonder why this wasn't failing Flow before?

Maybe the newer version catches more things?

Reply:
Maybe I missed to commit with-out `--no-verify`. I'm not always running flow. It's a lot faster to commit :-)

Is it possible to add a flow check to CircleCI? That would be great so I could do a quick commit if required and also see if everything is right. Maybe we could also remove the pre-commit check as it will be always checked. I'm still having troubles with VS code flow extension - I have to call `flow stop` before comitting.

Reply:
We could change the pre-commit hook to a pre-push hook if that'd make your life easier!

And yeah, would be good to add this to CI. I'll create a ticket

Reply:
Yes, `pre-push` would make it easier so I only have to remember that if I have to do a quick WIP push I have to add `no-verify`. 
We could change that as well with the CI issue.